### DISCUSSION TITLE
AccessDenied Error when using Next/Image

### DISCUSSION THREAD

**Discussion by Shadrack1701** [vercel/next.js#23418] [Help]
### What version of Next.js are you using?

latest

### What version of Node.js are you using?

AWS Lambda Node.js 12.x

### What browser are you using?

Brave

### What operating system are you using?

Windows

### How are you deploying your application?

Serverless NextJs (https://github.com/serverless-nextjs/serverless-next.js)

### Describe the Bug

When trying to display an image using Next/Image I am getting the error 
    `"url" parameter is valid but upstream response is invalid`

### Expected Behavior

The next/image component to return a smaller optimized image based on the parameters supplied.

### To Reproduce

You can see the symptoms by going to my dev site https://dev.cardscribed.com/about.

The actual erroring url is: `https://dev.cardscribed.com/_next/image?url=%2Fmascots.jpg&w=1920&q=75'
You can see the image is accessible by navigating to: `https://dev.cardscribed.com/mascots.jpg`

I opened an issue with serverless-nextjs [here](https://github.com/serverless-nextjs/serverless-next.js/issues/965).  I have a hunch it is an issue with my s3/lambda access configuration, but I wanted to open this here to see if there was some additional insight I could gather.  I haven't been able to see many issues similar to mine on the internet so hoping maybe someone here will have had a similar experience and can help me.

Any other information that would be helpful please let me know!

Thanks!
Matt Trachsel

**Comment by Djuki** [comment]
Same issue on my end:

S3 permissions set up like this:

```{
    "Version": "2012-10-17",
    "Id": "http referer policy example",
    "Statement": [
        {
            "Sid": "Allow get requests originating from domainss",
            "Effect": "Allow",
            "Principal": "*",
            "Action": [
                "s3:GetObject",
                "s3:GetObjectVersion"
            ],
            "Resource": "arn:aws:s3:::web/*",
            "Condition": {
                "StringLike": {
                    "aws:Referer": [
                        "http://localhost:3000/*",
                        "http://localhost:4000/*"
                    ]
                }
            }
        }
    ]
}

```


Image called:

```
<Image src={`s3img/home-page/protect.png`} width="800" height="470" alt="Protect" />
```

Request headers:

```Accept: image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8
Accept-Encoding: gzip, deflate
Accept-Language: sr-RS,sr;q=0.9,en-US;q=0.8,en;q=0.7
Connection: keep-alive
Cookie: ai_user=SgLot+CfBXPOLFlzqoWS7z|2021-12-03T08:59:37.604Z
Host: lbweb.test
Referer: http://lbweb.test/
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36

```

Getiing error:

```
"url" parameter is valid but upstream response is invalid
```

**Comment by alnaranjo** [comment]
Any update on this? I'm experiencing the same on next 13.4.7. Image is accessible on url via cloudformation but fails when params are provided
